src/pages/index.js

```diff
@@ -133,7 +133,7 @@ function handleCardLike(evt, dataId) {
   api
     .toggleLike(dataId, isLiked)
     .then(() => evt.target.classList.toggle("card__like-button_liked"))
-    .catch(console.error);
+    .catch(() => console.error);
```

Comment:
` .catch(console.error);` should be used instead of `.catch(() => console.error)`.  


It should be either `.catch(() => console.error());` or ` .catch(console.error);`, but not  `.catch(() => console.error)` because the `console.error` is not called inside the arrow function, so it doen't work


**Please, correct it everywhere in the code**